app/components/candidate_interface/incomplete_new_references_section_component.rb

```diff
@@ -0,0 +1,7 @@
+module CandidateInterface
+  class IncompleteNewReferencesSectionComponent < IncompleteSectionComponent
+    include ViewHelper
+
+    def message; end
```

Comment:
This null-method is a bit confusing. Is it needed to override an inherited method? Maybe add a comment?